package/MDAnalysis/coordinates/core.py

```diff
@@ -75,26 +88,30 @@ def reader(filename, **kwargs):
     All other keywords are passed on to the underlying Reader classes; see
     their documentation for details.
 
+    Parameters
+    ----------
+    filename : str
```

Comment:
`filename` can also be a `tuple`; this should be indicated in its parameter doc.